src/Microsoft.NET.Sdk.Aspire/Microsoft.NET.Sdk.Aspire.csproj

```diff
@@ -5,6 +5,9 @@
   <PropertyGroup>
     <PackageId>$(PackageId).Manifest-$(DotNetAspireManifestVersionBand)</PackageId>
     <Description>.NET Aspire workload manifest</Description>
+
+    <!-- this allows installing the manifest as a nuget, which is useful for testing -->
+    <PackageType />
```

Comment:
Just Curious, does this change anything in the nuspec?

Reply:
Yes, this removes:
```xml
    <packageTypes>
      <packageType name="DotnetPlatform" />
    </packageTypes>
```
.. from the `nuspec`. I checked other manifest nugets and they do have this specified, but I skipped this for the mono/wasm manifests, and it hasn't been an issue.

In a follow up for `dotnet/runtime`, I can change the way the nuget is installed which would remove the need for `PackageType` to be `''`.

Reply:
mmm, I suppose this property is set via Arcade. It would be good to check what the implications of that are with someone from coreeng team.